packages/wdio-mocha-framework/tests/adapter.test.ts

```diff
@@ -181,7 +181,14 @@ test('prepareMessage', async () => {
     expect(result.type).toBe('afterTest')
     expect(result.title).toBe('foobar')
     expect(result.file).toBe('/foo/bar.test.js')
-})
+
+    await new Promise((r) => setTimeout(r, 5000))
```

Comment:
That is not necessary, just mock the `_suiteStartDate` property to be `Date.now() - 3000`